ux/output.go

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2019-2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package ux
+
+import (
+	"fmt"
+
+	"github.com/ava-labs/avalanchego/utils/logging"
+	"go.uber.org/zap"
+)
+
+func Print(log logging.Logger, msg string, args ...zap.Field) {
```

Comment:
I think received zap args in ux function may not be convenient, we want to keep full control of the printed msg.
Probably will be better something on this line (given args ...interface{})
```
fmtMsg := fmt.Sprintf(msg. args...)
fmt.Println(fmtMsg)
log.Infof(fmtMsg)
```